.changeset/shaggy-socks-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixed view transition issue on webKit browsers that prevented scrolling to #fragments 
```

Comment:
```suggestion
Fixes a view transition issue on webKit browsers that prevented scrolling to #fragments 
```

Reply:
had a good trainer. I'm sure I'll get the times right at some point :)